src/theme/palette.ts

```diff
@@ -0,0 +1,3 @@
+const palette = {};
```

Comment:
export const palette, you have to specify types for the palette object, to know it's type you need to go to the theme file with createTheme function, press the command, and click on the palette key

Reply:
resolved